knowledgebase/importing-geojason-with-nested-object-array.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Importing GeoJSON with a deeply nested object array
+description: “Importing GeoJSON with a deeply nested object array“
+date: 2024-12-18
+---
+
+# Importing GeoJSON with a deeply nested object array
+
+### Question 
+How do I import GeoJSON with a nested object array?
+
+### Answer
+For this tutorial, we will use open data publicly available [here](https://opendata.esri.es/datasets/ComunidadSIG::municipios-ign/explore?location=39.536006%2C-0.303882%2C6.57).
```

Comment:
Will this link expire or change? It looks very specific, perhaps we could recommend they go to a specific website rather than this exact location URL.

Reply:
This link comes from a real use case we discussed on the Community Slack channel. To ensure reproducibility and avoid relying on the URL, I recommend downloading the GeoJSON and making it available in our public bucket, similar to how we provide the `New York Taxi Data` [here](https://clickhouse.com/docs/en/getting-started/example-datasets).